sdk/compute/Microsoft.Azure.Management.Compute/src/Microsoft.Azure.Management.Compute.csproj

```diff
@@ -6,12 +6,12 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Compute</PackageId>
     <Description>Provides developers with libraries for the updated compute platform under Azure Resource manager to deploy virtual machine, virtual machine extensions and availability set management capabilities. Launch, restart, scale, capture and manage VMs, VM Extensions and more. Note: This client library is for Virtual Machines under Azure Resource Manager.</Description>
-    <Version>29.1.0</Version>
+    <Version>29.2.0</Version>
     <AssemblyName>Microsoft.Azure.Management.Compute</AssemblyName>
     <PackageTags>management;virtual machine;compute;</PackageTags>
     <PackageReleaseNotes>
       <![CDATA[
-        This is a public release of the Azure Compute SDK. Included with this release is adding $expand option for Availability Set List by Subscription API.
+        This is a public release of the Azure Compute SDK. Included with this release is adding Automatic Repairs Policy for Virtual Machine Scale Sets.
```

Comment:
Add more features by other Swagger updates.
- ScaleInPolicy for VMSS
- requireGeustProvisionSignal property in OsProfile.

Reply:
Updated.